### DISCUSSION TITLE
看不懂识别结果, 感觉有问题 求解答

### DISCUSSION THREAD

**Discussion by yanchujian** [PaddlePaddle/PaddleOCR#12361] [Q&A]
使用代码
from paddleocr import PaddleOCR, draw_ocr

# Paddleocr目前支持的多语言语种可以通过修改lang参数进行切换
# 例如`ch`, `en`, `fr`, `german`, `korean`, `japan`
ocr = PaddleOCR(use_angle_cls=True, lang="ch")  # need to run only once to download and load model into memory

img_path='./ppocr_img/imgs/11.jpg'
result = ocr.ocr(img_path, cls=True)
for idx in range(len(result)):
    res = result[idx]
    for line in res:
        print(line) 

得出结果如下:
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/41255670/5a045c00-1573-465f-afbb-f4414851604b)
使用的测试图片为:
![11](https://github.com/PaddlePaddle/PaddleOCR/assets/41255670/6fc96824-e2d5-4658-a3d1-98989fb03a54)
求大佬解释一下
然后使用自己的图片测试,返回为空:
![11](https://github.com/PaddlePaddle/PaddleOCR/assets/41255670/1be0e337-1cc0-42c0-a5a4-cf76204b63b6)

**Comment by tink2123** [comment]
确认一下系统是否支持中文字符集？

**Comment by yanchujian** [comment]
> 确认一下系统是否支持中文字符集？

用命令行的方式可以正确显示中文 这是什么原因呢 同时下面一张图片也能正确识别出来.
